canonical/vanilla-framework
# Fix figma library alert

## Done
Fixed the Figma library alert which [is failing](https://github.com/canonical/vanilla-framework/actions/runs/9552881621/job/26330491088). It seems like the issue was with the manually added newlines and json encoding.

## QA

- But the release notes you want to test with in a release.json like so: 
```
{
  "release": {
    "body": "## Getting Vanilla Framework\r\n\r\nInstall with NPM: https://www.npmjs.com/package/vanilla-framework\r\nVisit the documentation at https://vanillaframework.io/docs\r\n\r\n## New in Vanilla v4.13.0\r\n\r\n## \uD83D\uDE80 Features\r\n\r\n- Add new image container component and deprecate legacy image variants by @jmuzina (#5097)\r\n\r\n## \uD83D\uDC1B Bug Fixes\r\n\r\n- Fix ascending/descending icons on sortable tables by @pastelcyborg (#5135)\r\n- Fix application layout tooltip overflow by @pastelcyborg (#5134)\r\n\r\n## \uD83D\uDCDD Documentation\r\n\r\n- Update Pull Requests README with additional information and steps by @pastelcyborg (#5137)\r\n\r\n## \uD83D\uDD28 Maintenance\r\n\r\n- Update accordion pattern to use new color theme variables by @jmuzina (#5138)\r\n- Update Matrix component with new theme color variables by @pastelcyborg (#5136)\r\n- Notify Figma library maintainers on release by @dgtlntv (#5132)\r\n- Update dependency @canonical/cookie-policy to v3.6.4 by @renovate (#5131)\r\n- Update article pagination to use new theme variables by @jmuzina (#5126)\r\n- Update lists pattern to use new color theme variables by @jmuzina (#5124)\r\n- Update Pull Quote component with new theme color variables by @pastelcyborg (#5133)\r\n- Update alternative background colour in dark theme by @bartaz (#5128)\r\n- Update card pattern to support color themes by @jmuzina (#5112)\r\n- Bump braces from 3.0.2 to 3.0.3 by @dependabot (#5121)\r\n- Update Table of Contents to use new color theme variables by @pastelcyborg (#5113)\r\n- Reduce padding on navigation items by @bartaz (#5116)\r\n\r\n**Full Changelog**: https://github.com/canonical/vanilla-framework/compare/v4.12.0...v4.13.0"
  }
}
```
- Run the following act command: `act --job notify-figma -e release.json --container-architecture linux/amd64 -s FIGMA_LIBRARY_ALERT_URL="the-webbot-url"`
- See if that runs without errors and posts a message to the Figma library maintainers chat
